src/inverseRobot.js

```diff
@@ -24,7 +24,25 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const keysA = Object.keys(robot);
+  const valuesA = Object.values(robot);
+  const arr = [ ...valuesA ];
+
+  for (let i = 0; i < arr.length; i++) {
+    const shifted = arr.shift();
+
+    if (arr.includes(shifted)) {
+      return null;
+    }
+  }
+
+  const result = {};
```

Comment:
[NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)